lib/msal-node/src/client/PublicClientApplication.ts

```
@@ -271,6 +278,13 @@ export class PublicClientApplication
             return this.nativeBrokerPlugin.acquireTokenSilent(brokerRequest);
         }
 
+        if (request.redirectUri) {
+            this.logger.warning(
+                "RedirectUri provided in an unsupported scenario. Falling back to default"
```

Same comment here, rather than ignoring I prefer we throw

---

Changed to throw instead